db/migrate/20200912194928_create_places.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePlaces < ActiveRecord::Migration[6.0]
+  def change
+    create_table :places do |t|
+      t.text :address
+      t.string :name
+      t.integer :latitude
+      t.integer :longitutude
```

Comment:
please change to longitude 
(typo)

Reply:
So because I git put from... My changes and correction about the table are not here. I ve made all this correction yesterday before. I will do it again nevernind nut I feel that we shold all start fromthe same master ay the same time